hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.controller.js

```diff
@@ -22,6 +22,10 @@ define([
     settings, pubSub) {
     $log.debug('Controller: ModalContractNewController');
 
+    var providersKey = {
```

Comment:
treat it as a constant, and maybe add a inline comment explaining what is the key and what is the value

```js
// <explanation-of-key>: <explanation-of-value>
var PROVIDERS = {
  'hrjc_health_insurance_provider': 'health_insurance_provider'
}
```

Reply:
Good thought 👍